thentos-adhocracy/src/Thentos/Adhocracy3/Backend/Api/Simple.hs

```diff
@@ -452,6 +486,9 @@ extractUserId resp = do
         (Aeson.eitherDecode . Client.responseBody $ resp :: Either String TypedPath)
     userIdFromPath $ tpPath resource
 
+sendRequest :: Client.Request -> IO (Client.Response LBS)
+sendRequest req = Client.withManager Client.defaultManagerSettings $ Client.httpLbs req
```

Comment:
I don't know, I think you made the refactoring to use a single manger so you should know how to do that. Maybe you can change that code in a separate PR? It wouldn't belong in this PR anyway, as that code is old(er) -- I just moved it around a bit.